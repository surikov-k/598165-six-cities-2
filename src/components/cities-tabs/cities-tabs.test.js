import React from 'react';
import renderer from 'react-test-renderer';

import CitiesTabs from './cities-tabs.jsx';

describe(`<CitiesTabs/>`, () => {
  it(`should render correctly`, () => {
    const component = renderer
      .create(<CitiesTabs
        currentCity={``}
        cities={[]}
        onChangeCity={() => {}}
        active={0}
      />)
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
