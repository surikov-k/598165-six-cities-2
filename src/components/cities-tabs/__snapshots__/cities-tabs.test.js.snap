// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CitiesTabs/> should render correctly 1`] = `
<div
  className="tabs"
>
  <section
    className="locations container"
  >
    <ul
      className="locations__list tabs__list"
    />
  </section>
</div>
`;
